proposals/semi-auto-properties.md

```diff
@@ -12,114 +12,85 @@ In these cases by now you always have to create an instance field and write the
 
 ## Specification changes
 
-The following changes are to be made to [§14.7.4](https://github.com/dotnet/csharpstandard/blob/draft-v6/standard/classes.md#1474-automatically-implemented-properties):
+[§14.7.1](https://github.com/dotnet/csharpstandard/blob/standard-v6/standard/classes.md#1471-general) *Properties - General*
+
+> A *property_initializer* may only be given for ~~an automatically implemented property, and~~ **a property that has a backing field that will be emitted and the property either does not have a setter, or its setter is auto-implemented. The *property_initializer*** causes the initialization of the underlying field of such properties with the value given by the *expression*.
+
+[§14.7.4](https://github.com/dotnet/csharpstandard/blob/standard-v6/standard/classes.md#1474-automatically-implemented-properties) *Automatically implemented properties*
+
+> An automatically implemented property (or ***auto-property*** for short), is a non-abstract non-extern
+> property with ~~semicolon-only accessor bodies. Auto-properties must have a get accessor and can optionally~~
+> ~~have a set accessor.~~ **either or both of:**
+> 1. **an accessor with a semicolon-only body**
+> 2. **usage of the `field` contextual keyword within the accessors or**
+>    **expression body of the property. The `field` identifier is only considered the `field` keyword when there is**
+>    **no existing symbol named `field` in scope at that location.**
+> 
+> When a property is specified as an ~~automatically implemented property~~ **auto-property**, a hidden **unnamed** backing field is automatically
+> available for the property ~~, and the accessors are implemented to read from and write to that backing field~~.
+> **For auto-properties, any semicolon-only `get` accessor is implemented to read from, and any semicolon-only**
+> **`set` accessor to write to its backing field.**
+> 
+> **The backing field can be referenced directly using the `field` keyword**
+> **within all accessors and within the property expression body. Because the field is unnamed, it cannot be used in a**
+> **`nameof` expression.**
+> 
+> If the auto-property has ~~no set accessor~~ **only a semicolon-only get accessor**, the backing field is considered `readonly` ([§14.5.3](https://github.com/dotnet/csharpstandard/blob/standard-v6/standard/classes.md#1453-readonly-fields)).
+> Just like a `readonly` field, a getter-only auto property **(without a set accessor or an init accessor)** can also be assigned to in the body of a constructor
+> of the enclosing class. Such an assignment assigns directly to the ~~readonly~~ backing field of the property.
```

Comment:
i'd prefer to say that `get;` is shorthand for `get => field;` and if there is no setter then the backing field is readonly if the getter doesn't mutate `field`.  It is not readonly if it does.